### PR TITLE
[feature] Test llama-7b-gptq on scheduler_rolling_batch

### PR DESCRIPTION
**Pull request by KexinFeng**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Review by lanking520 — COMMENTED** [review]
could you add this test to our CI testing pipelines?

File: tests/integration/llm/prepare.py [hunk 1]
```diff
@@ -380,6 +380,12 @@
         "option.max_rolling_batch_size": 4,
         "option.tensor_parallel_degree": 4,
         "load_on_devices": 0
+    },
+    "llama2-7b-chat-gptq": {
+        "option.model_id": "TheBloke/Llama-2-7b-Chat-GPTQ",
+        "engine": "Python",
+        "option.max_rolling_batch_size": 4,
```
Comment (lanking520) [inline tests/integration/llm/prepare.py:387]: do we need this settings for scheduler?

File: .github/workflows/rolling_batch_integration.yml [hunk 2]
```diff
@@ -85,6 +85,15 @@ jobs:
           serve -m test=file:/opt/ml/model/test/
           python3 rb_client.py scheduler_single_gpu bloom-560m
           docker rm -f $(docker ps -aq)
+      - name: Test llama2-7b-chat-gptq
+        working-directory: tests/integration
+        run: |
+          rm -rf models
+          python3 llm/prepare.py rolling_batch_scheduler llama2-7b-chat-gptq
```
Comment (lanking520) [inline .github/workflows/rolling_batch_integration.yml:92]: will this test work if we didn't install `autogptq` and `optimum` ?

File: tests/integration/llm/prepare.py (hunk 1 above)
Comment (lanking520) [reply tests/integration/llm/prepare.py:387]: I don't think this is necesary. If you specify rolling batch

File: .github/workflows/rolling_batch_integration.yml (hunk 2 above)
Comment (lanking520) [reply .github/workflows/rolling_batch_integration.yml:92]: You can refer to this: https://github.com/deepjavalibrary/djl-serving/blob/master/tests/integration/llm/prepare.py#L649

Comment (lanking520) [reply .github/workflows/rolling_batch_integration.yml:92]: you can pass with requirements.txt to install those dependencies on the fly

File: tests/integration/llm/prepare.py (hunk 1 above)
Comment (lanking520) [reply tests/integration/llm/prepare.py:387]: This is a different error, the name you give to the model is wrong

File: tests/integration/llm/prepare.py [hunk 3]
```diff
@@ -380,6 +380,12 @@
         "option.max_rolling_batch_size": 4,
         "option.tensor_parallel_degree": 4,
         "load_on_devices": 0
+    },
+    "llama2-7b-chat-gptq": {
```
Comment (lanking520) [inline tests/integration/llm/prepare.py:384]: you are now putting them in a CI, but not installing the necessary wheel for the test. How this will work?

**Review by lanking520 — CHANGES_REQUESTED** [review]
Please make sure the CI test are passable

**Review by lanking520 — APPROVED** [review]
